### DISCUSSION TITLE
After enabling the lang.markdown LazyExtra, getting Mason error about locked file

### DISCUSSION THREAD

**Discussion by rykugur** [LazyVim/LazyVim#3653] [Q&A]
I recently enabled the `lang.markdown` LazyExtra, and ever since when I open my first file in neovim I get the following error:

```
[ERROR Fri 14 Jun 2024 11:57:35 AM CDT] ...e/nvim/lazy/mason.nvim/lua/mason-core/installer/init.lua:249: Installation failed for Package(name=marksman) error="Lockfile exists, installation is already running in another process (pid: 108486). Run with :MasonInstall --force to bypass."
```

I run nixOS, and use home-manager (although I do _not_ have my neovim configured in home-manager, I just use HM to symlink my config files into `~/.config/nvim` (using mkOutOfStoreSymlink at the moment)). Up until now I've had zero issues with Mason and other LSP's.

Has anybody else run into this issue?

EDIT: `:MasonInstall --force` doesn't seem to have any effect.

**Comment by rubiin** [comment]
you could try to kill the process `108486` and do `:MasonInstall --force`

**Reply by rykugur** [reply]
> you could try to kill the process `108486` and do `:MasonInstall --force`

I did indeed already try this - that pid does not exist. It's always the same pid, too, even after a reboot; and the same error/pid on my other computer.

Likely a nixOS thing

**Reply by tleveille** [reply]
I had the same issue on a fedora, solved it by removing the lock files in ~/.local/share/nvim/mason/staging.

It probably happened because I was sharing my homedir with a VM and used neovim on both at the same time. Quite a dangerous thing to do I guess.

**Reply by Mo0rBy** [reply]
This works! Thx @tleveille!

**Reply by wangyaodream** [reply]
@tleveille Thanks! This worked for me too.
